modules/custom/openy_campaign/src/Form/MemberEmailEditForm.php

```diff
@@ -30,6 +30,8 @@ public function buildForm(array $form, FormStateInterface $form_state, $member_i
     $form['#prefix'] = '<div class="container">';
     $form['#suffix'] = '</div>';
 
+    // The block is rendered for each user separately.
+    // We can't cache it.
```

Comment:
we have user cache tags, or user context. https://www.drupal.org/docs/8/api/cache-api/cache-contexts
Let's work on this after release.